homeassistant/components/zwave_js/config_flow.py

```diff
@@ -309,6 +332,44 @@ async def async_step_user(
 
         return await self.async_step_manual()
 
+    async def async_step_usb(self, discovery_info: dict[str, str]) -> FlowResult:
+        """Handle USB Discovery."""
+        if not is_hassio(self.hass):
+            return self.async_abort(reason="discovery_requires_supervisor")
+
+        vid = discovery_info["vid"]
+        pid = discovery_info["pid"]
+        serial_number = discovery_info["serial_number"]
+        device = discovery_info["device"]
+        manufacturer = discovery_info["manufacturer"]
+        description = discovery_info["description"]
+        # The Nortek sticks are a special case since they
+        # have a Z-Wave and a Zigbee radio. We need to reject
+        # the Zigbee radio.
+        if (
+            vid == "10C4"
+            and pid == "8A2A"
+            and "Z-Wave" not in discovery_info["description"]
+        ):
+            return self.async_abort(reason="not_zwave_device")
+
+        await self.async_set_unique_id(
+            f"{vid}:{pid}_{serial_number}_{manufacturer}_{description}"
```

Comment:
Won't this allow an overlap with other config entries that use the home id as unique id?

Reply:
I'll make sure its gets overwritten with the home id before it finishes in the final version

Reply:
Ok, so we won't be able to create a duplicate config entry (good), but the USB discovery flow for a device won't automatically close when creating a config entry for that device from another flow source (less good).